### PR TITLE
Completed prep-javascript-operators-and-expressions exercise.

### PR DESCRIPTION
**Pull request by Edward-J-Kim**
save

### CODE REVIEW

File: prep-javascript-operators-and-expressions/main.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+var number1 = 5;
```
Comment (Zhiwutian) [inline prep-javascript-operators-and-expressions/main.js:1]: Hey Edward, make sure to convert all of the `var` declarations to `const` we won't be using `var` in this class.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Looking good, just a couple updates are noted for completion

File: prep-javascript-operators-and-expressions/prep-javascript-operators-and-expressions-quiz-notes.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# prep-javascript-operators-and-expressions-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
```
[the remark this answers is not in the document]
Comment (Edward-J-Kim) [reply prep-javascript-operators-and-expressions/prep-javascript-operators-and-expressions-quiz-notes.md:6]: answered the quiz questions.

File: prep-javascript-operators-and-expressions/main.js (hunk 1 above)
Comment (Edward-J-Kim) [reply prep-javascript-operators-and-expressions/main.js]: converted all.

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**